src/couch_peruser/test/couch_peruser_test.erl

```diff
@@ -212,6 +240,32 @@ should_delete_user_db(TestAuthDb) ->
     wait_for_db_delete(UserDbName),
     ?_assert(not lists:member(UserDbName, all_dbs())).
 
+should_delete_user_db_with_custom_prefix(TestAuthDb) ->
+    User = "bar",
+    UserDbName = <<"newuserdb-626172">>,
+    set_config("couch_peruser", "delete_dbs", "true"),
+    set_config("couch_peruser", "database_prefix", "newuserdb-"),
+    create_user(TestAuthDb, User),
+    wait_for_db_create(UserDbName),
+    ?assert(lists:member(UserDbName, all_dbs())),
```

Comment:
**This is a common mistake**: `should_delete_user_db_with_custom_prefix ` is eunit _generator_ and this assertion here not executed during the test, but during the generation, i.e. in case it fail and throw exception (as `?assert` does) it'll kill generator and test will be just quietly skipped.

What you actually need to do here is to catch result of `lists:member/1`, e.g. `AfterCreate = lists:member(UserDbName, all_dbs())`, do then same below and then return a list of assertion functions: `[?_assert(AfterCreate), ?_assertNot(AfterDelete)]`.

This is also true for the next test `should_delete_user_db_with_custom_special_prefix`

Reply:
Ah, I see you've just cut'n'paste bad tests that already in this suite.
Here is the fix for that, check it out: #1111

Reply:
Fixed in commit 193cf6d